hamocc/dipowa.F90

```diff
@@ -190,16 +186,8 @@ subroutine dipowa(kpie,kpje,kpke,omask,lspin)
 
   if(.not. lspin) THEN
 ! sediment ocean interface
-!
-! CAUTION - the following assumes same indecees for ocetra and powtra
-!           test npowa_base 071106
-!           check mo_param1_bgc.f90 for consistency
   do iv = 1, npowtra
-     iv_oc = iv
-#ifdef cisonew
-     if (iv == ipowc13) iv_oc=isco213
-     if (iv == ipowc14) iv_oc=isco214
-#endif
+     iv_oc = map_por2octra(iv)
      do i = 1,kpie
         l = 0
         if (omask(i,j) > 0.5) then
```

Comment:
See https://github.com/NorESMhub/BLOM/issues/191#issuecomment-1238443457 - since it seems as it doesn't affect CMIP backwards compatibility at all, I would then suggest to do it in one goal and exchange the indices. The code as is was tested against the former master.